### PR TITLE
Revert config to defaults

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

File: tests/data/mock_manifests/mock_example_biospecimen_manifest.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Sample ID,Patient ID,Tissue Status,Component
-1,1,Healthy,Biospecimen
+Sample ID,Patient ID,Tissue Status,Component,Id,entityId
+1,1,Healthy,Biospecimen,61a70a22-f26d-46b1-8f6f-29ed54658a58,
```
Comment (BryanFauble) [inline tests/data/mock_manifests/mock_example_biospecimen_manifest.csv:2]: Was this change intentional?

File: tests/data/mock_manifests/mock_example_bulkrnaseq_manifest.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Filename,Sample ID,File Format,Component,Genome Build,Genome FASTA,entityId
-Manifest Submission - Manual test - file-based manifest submission/test-dataset/sample A.txt,1,FASTQ,BulkRNA-seqAssay,,,syn63561932
\ No newline at end of file
+Filename,Sample ID,File Format,Component,Genome Build,Genome FASTA,entityId,Id
+Manifest Submission - Manual test - file-based manifest submission/test-dataset/sample A.txt,1,FASTQ,BulkRNA-seqAssay,,,syn63561932,1453e4c5-9847-4527-b70b-c8b8f5cd89a7
```
Comment (BryanFauble) [inline tests/data/mock_manifests/mock_example_bulkrnaseq_manifest.csv:2]: Was this change intentional?

**Review by BryanFauble — APPROVED** [review]

**Comment by thomasyu888** [comment]
Thoughts on either

1. double merge and not wait for tests to run on Bryan's PR.
2. Change base to develop on this PR

@andrewelamb / @BryanFauble
